.storybook/preview-head.html

```diff
@@ -293,4 +293,23 @@
     background-clip: content-box;
   }
 
+  .importPath {
+    color: #333;
+    font-size: 14px;
+    cursor: pointer;
+    font-family: "Operator Mono","Fira Code Retina","Fira Code","FiraCode-Retina","Andale Mono","Lucida Console",Consolas,Monaco,monospace;
+  }
+
+  .importPath--highlight {
+    color: #006699;
+  }
+
+  .importPath:hover, .importPath:hover svg {
+    color: darkorange;
+  }
+
+  .importPath:hover .importPath--highlight {
+    color: orange;
+  }
+
```

Comment:
To note: I'm going to start specifically not using FDS values in here. I want all the stuff in Preview to be more generalized, in the off chance that we want to share some of this stuff back to the Storybook community.

Reply:
makes sense - I was _just_ about to suggest `var(--font-family-mono)`, but your approach makes sense 👍